HACKTOBERFEST.md

```diff
@@ -0,0 +1,32 @@
+# :pizza: Hacktoberfest 2019 :beer:
+
+It's that time of the year again!
+[Hacktoberfest](https://hacktoberfest.digitalocean.com) is here, and `gensim` needs **your** help.
+We've prepared a list of good issues to work on: [gensim hacktoberfest issues](https://github.com/RaRe-Technologies/gensim/labels/hacktoberfest).
+
+If the learning curve for `gensim` is a bit steep, give the [smart_open](https://github.com/RaRe-Technologies/smart_open) repository a try.
+`smart_open` is an important dependency of `gensim`: it performs file I/O over a variety of protocols and formats.
+There's also a list of Hacktoberfest-friendly issues to work on: [smart_open hacktoberfest issues](https://github.com/RaRe-Technologies/smart_open/labels/hacktoberfest).
+
+Of course, we welcome contributions on any of the existing issues, not just the ones labeled `hacktoberfest`.
+Just be sure that no-one else is already working on it first.
```

Comment:
This is a little vague, and may result in people either (1) hesitating to make a small fix (by feeling they must ask, "is anyone working on this?" first), or (2) prematurely claiming an issue without follow-through (by declaring "I'm working on this" then disappearing for an arbitrary amount of time). 

Perhaps instead: 

> If an issue is simple & quick, you can just submit your PR, with a proper reference to the issue it addresses. 
> 
> If an issue requires a little more work, but you have a good idea of how to proceed & know when you'll be submitting some initial work, please post a short note about your plans to the issue, or a "work-in-progress" ("[WIP]") pull-request indicating work is underway, to help avoid wasted duplicate work.